src/components/UploadCSVView/CSVViewTable/CSVViewTable.jsx

```diff
@@ -316,59 +321,66 @@ const CSVViewTable = ({ rows, boxNumberMap }) => {
         </div>
       )}
       {isMobile && (
-        <Accordion allowToggle>
-          <Flex flexDirection="column" gap="20px">
-            {page.map((rowData, index) => {
-              prepareRow(rowData);
-              return (
-                <AccordionItem
-                  key={rowData.original.id}
-                  borderWidth="1px"
-                  borderRadius="8px"
-                  borderColor={rowData.values.error && 'red'}
-                >
-                  {({ isExpanded }) => (
-                    <>
-                      <h2>
-                        <AccordionButton
-                          _expanded={{ bg: 'white' }}
-                          onClick={() => {
-                            setEditId(null);
-                          }}
-                        >
-                          <Box flex="1" textAlign="left" fontWeight="bold">
-                            {!isExpanded ? `Box #${rowData.values.boxNumber}` : ''}
-                          </Box>
-                          <AccordionIcon />
-                        </AccordionButton>
-                      </h2>
-                      <AccordionPanel pb={4}>
-                        {editId === rowData.original.id ? (
-                          <EditableRow
-                            editFormData={editFormData}
-                            handleEditFormSubmit={handleEditFormSubmit}
-                            isError={rowData.values.error}
-                            boxNumberMap={boxNumbers}
-                            updateBoxNumberMap={updateBoxNumberMap}
-                            lineNumber={pageIndex * 10 + index + 1}
-                            handleDeleteRow={handleDeleteRow}
-                          />
-                        ) : (
-                          <ReadOnlyRow
-                            data={rowData}
-                            editRow={editRow}
-                            handleDeleteRow={handleDeleteRow}
-                            isError={rowData.values.error}
-                          />
-                        )}
-                      </AccordionPanel>
-                    </>
-                  )}
-                </AccordionItem>
-              );
-            })}
-          </Flex>
-        </Accordion>
+        <div className={styles['csv-table-container-mobile']}>
+          <Accordion {...accordionProps}>
+            <Flex flexDirection="column" gap="20px">
+              {page.map((rowData, index) => {
+                prepareRow(rowData);
+                return (
+                  <AccordionItem
+                    key={rowData.original.id}
+                    borderWidth="1px"
+                    borderRadius="8px"
+                    borderColor={rowData.values.error && 'red'}
```

Comment:
Can we use the warning color in [vars.css](https://github.com/ctc-uci/find-your-anchor-frontend/blob/dev/src/common/vars.css) instead of red?